src/seedu/addressbook/data/person/Address.java

```diff
@@ -29,6 +29,14 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
     }
 
     /**
+     *  Method to get printable string
+     */
```

Comment:
Coding standard violation:exclamation: The first statement of the header comment does not follow the prescribed style. See point 4, 3rd bullet point of the [[Java Coding Standard: Comments](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#comments)]

Reply:
Thanks Damith. I will take not of this in the future.